katas/content/superdense_coding/bob_decodes_message/index.md

```diff
@@ -0,0 +1,6 @@
+**Inputs:**
+
+1. `qAlice` : Qubit received from Alice.
+2. `qBob` : Bob's part of the entangled pair.
+
+**Goal** :  Decode the message using the qubit received from Alice and reset both qubits to a $\ket{00}$ state. For this, retrieve two bits of classic data from the qubits and return a tuple of the form `(Bool, Bool)` to represent `message`. The state of the qubits in the end of the operation should be $\ket{00}$.
```

Comment:
We need a note on testing here as well. Also, since we've modified the testing harness, there is no requirement to reset qubits to 0, the solution passes without that

```suggestion
**Goal** :  Decode the message using the qubit received from Alice. For this, retrieve two bits of classic data from the qubits and return a tuple of the form `(Bool, Bool)` to represent `message`. The state of the qubits in the end of the operation does not matter.

Note that your solution is tested as part of an end-to-end implementation of the protocol; the goal is to get the message transmitted correctly.
```